local-cli/automated-vs/utils/msbuild.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const MSBUILD_VERSIONS = ['15.0', '14.0', '12.0', '4.0'];
+
```

Comment:
Is there an automated way to grab the available / possible versions (from registry or otherwise)?

Reply:
yes, did that for my next pr